### PR TITLE
News: 2021-09-06

### PR DESCRIPTION
**Pull request by dev-yakuza**
See:
- https://github.com/sider/runners/releases/tag/0.53.4
- https://github.com/sider/sider-docs/pull/678

### CODE REVIEW

**Comment by dev-yakuza** [comment]
@suetsuguk 
I think this PR should be merged after merging https://github.com/sider/sider-docs/pull/678 PR

**Comment by dev-yakuza** [comment]
https://deploy-preview-684--sider-docs.netlify.app/news/2021

File: docs/news/2021.md [hunk 1]
```diff
@@ -10,6 +10,33 @@ This document describes notable changes on Sider in the year 2021.
 
 ---
 
+## Add Metrics Runners filtering option
+
+<time className="NewsDate" dateTime="2021-09-06">September 06, 2021</time>
```
Comment (ybiquitous) [inline docs/news/2021.md:15]: [trivial]

```suggestion
<time className="NewsDate" dateTime="2021-09-06">September 6, 2021</time>
```

**Review by ybiquitous — APPROVED** [review]
LGTM 👍🏼

File: docs/news/2021.md (hunk 1 above)
Comment (dev-yakuza) [reply docs/news/2021.md]: Thanks!

File: docs/news/2021.md [hunk 2]
```diff
@@ -10,6 +10,33 @@ This document describes notable changes on Sider in the year 2021.
...
 ---
 
+## Add Metrics Runners filtering option
+
+<time className="NewsDate" dateTime="2021-09-06">September 6, 2021</time>
+
+Today, we have released the Metrics Runners filtering option! 🎉
+
+This option allows you to filter out certain files or directories from being analyzed by the metrics runners (`Metrics File Info`, `Metrics Complexity` and `Metrics Code Clone`).
+See the [document](../getting-started/custom-configuration.md#`linter.metrics.ignore`) for details.
```
Comment (suetsuguk) [inline docs/news/2021.md:20]:
```suggestion
See the [document](../getting-started/custom-configuration.md#linter_metrics_ignore) for details.
```

Be sure to check if this link is correct after merging PR #678.

File: docs/news/2021.md [hunk 3]
```diff
@@ -10,6 +10,33 @@ This document describes notable changes on Sider in the year 2021.
...
+
+In addition, this release includes the following tool updates: ✨
+
+| Tool                                            | Language     | Version             | Ref                                                                          |
+| ----------------------------------------------- | ------------ | ------------------- | ---------------------------------------------------------------------------- |
+| [Checkstyle](../tools/java/checkstyle.md)       | Java         | 8.45.1 → **9.0**    | [changes](https://checkstyle.org/releasenotes.html)                          |
+| [CoffeeLint](../tools/javascript/coffeelint.md) | CoffeeScript | 5.0.1 → **5.0.2**   | [changes](https://github.com/coffeelint/coffeelint/blob/v5.0.2/CHANGELOG.md) |
```
Comment (suetsuguk) [inline docs/news/2021.md:27]: Add line for `detekt 1.18.1`.

File: docs/news/2021.md (hunk 2 above)
Comment (dev-yakuza) [reply docs/news/2021.md:20]: I will check it again after merging PR!

File: docs/news/2021.md (hunk 3 above)
Comment (dev-yakuza) [reply docs/news/2021.md:27]: Thanks. I added it on https://github.com/sider/sider-docs/pull/684/commits/84afbd145f34efd94cc754b36e0ff57078661ed8

**Review by suetsuguk — APPROVED** [review]
LGTM. Thank you.

File: docs/news/2021.md (hunk 2 above)
Comment (dev-yakuza) [reply docs/news/2021.md]: the link was broken, so I fixed it on https://github.com/sider/sider-docs/pull/684/commits/773dd7f213b72b42942e2d428eefecb30d6de2d5

Thanks! 🙇‍♂️

**Pull request merged**